File: sach/de3369.cpp
//DE3369.CPP - Tran Huu Nam - hsgtin.vn - 2024
#include <iostream>
using namespace std;
int main() {
    int n, d, m, dem=0, x;
    cin>>n>>d>>m;
    for (int i=0; i<n; i++) {
        cin>>x;
        if (x%d==0 || x%m==0) dem++;
    }
    cout<<dem;
    return 0;
}

File: sach/de3369.py
//DE3369.PY - Tran Huu Nam - hsgtin.vn - 2024
import sys
def main():
    n, d, m, dem = 0, 0, 0, 0
    input_data = sys.stdin.read().split()
    n = int(input_data[0])
    d = int(input_data[1])
    m = int(input_data[2])
    for i in range(n):
        x = int(input_data[3 + i])
        if x % d == 0 or x % m == 0:
            dem += 1
    print(dem)
if __name__ == "__main__":
    main()

